Python/Product/Analysis/Analyzer/ImportStatementWalker.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
```

Comment:
Needs a copyright header (did you add this or just merge it in from someone else's PR? regardless, copy/paste a header please)